css/style.css

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Inter', sans-serif;
+}
+
+a:hover {
+  background-color: #36b37f;
+}
+
+.button {
+  height: 48px;
+  width: 91px;
+  border-radius: 4px;
+  border-color: #36b37f;
+  padding: 12px;
+}
+
+.button-enabled {
+  background-color: #36b37f;
+}
+
+.button:hover {
+  background-color: #36b37f;
+}
+
+.button:active {
+  background-color: #008552;
+}
+
+#mobile-navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 10px 12px;
+  width: 375px;
+}
+
+.welcome-logo {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.main-section {
+  background-image: url("../assets/backgrounds/header-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 400px;
+  height: 685px;
+  top: 92px;
+}
```

Comment:
- [ ] Your design is not centered on devices with `768px` screens as shown below 👇 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/164275646-a64298a6-1101-4e3d-a98d-070161ee8fc7.png)

### Figma Design
![image](https://user-images.githubusercontent.com/46839548/164275903-a1599f6d-09de-4797-962b-6bef3eb27ae0.png)

Reply:
I think the problem stems from the fact that you have `width: 400px`. This makes the container to be only `400px` wide. Try removing that line of code.

Reply:
@codepantha yes, Now it's fixed, and with the previous commit, git marked it as outdated also. Thanks for the suggestion

Reply:
You're welcome